javascript/src/main/components/BasicCourseSearch/CourseSearchFormInstructor.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import { Form, Button } from "react-bootstrap";
+//import fetch from "isomorphic-unfetch";
+
+const CourseSearchFormInstructor = ({ setCourseJSON, fetchJSON }) => {
+
+
+
+    const [startQuarter, setStartQuarter] = useState("20204");
+    const [endQuarter, setEndQuarter] = useState("20204");
+    const [instructorText, setInstructor]=useState("");
+    
+
+    const handleSubmit = (event) => {
+        //console.log(event);
+        event.preventDefault();
+        fetchJSON(event, { startQuarter, endQuarter, instructorText}).then((courseJSON) => {
+            setCourseJSON(courseJSON);
+        });
+    };
+
+    // FALL = 4, F
+    // WINTER = 1, W
+    // SPRING = 2, S
+    // SUMMER = 3, M
+    const handleStartQuarterOnChange = (event) => {
+        setStartQuarter(event.target.value);
```

Comment:
Don't need these comments anymore

Reply:
Normally, I'd agree with deleting comments (especially if it's commented out code), but I think in this case, these might be ok to keep.